docs/ci/codemagic.mdx

```diff
@@ -110,9 +110,22 @@ scripts:
     script: |
       # Install Shorebird
       curl --proto '=https' --tlsv1.2 https://raw.githubusercontent.com/shorebirdtech/install/main/install.sh -sSf | bash
-
-      # Add Shorebird to PATH
-      echo PATH="/Users/builder/.shorebird/bin:$PATH" >> $CM_ENV
```

Comment:
The better fix might just be to use $HOME.

Reply:
True 😀